scripts/integration-test-all.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# "stable" mode tests assume data is static
+# "live" mode tests assume data dynamic
+
+
+SCRIPT=`basename ${BASH_SOURCE[0]}`
+TEST=""
+BINARY="$PWD/bin/gossamer"
+
+PORT="7001"
+RPC_PORT="8545"
+IP_ADDR="0.0.0.0"
+HOST="http://$IP_ADDR:$PORT"
+HOST_RPC="http://$IP_ADDR:$RPC_PORT"
+MODE="stable"
+
+KEY="alice"
+
+usage () {
+  echo "Usage: $SCRIPT"
+  echo "Optional command line arguments"
+  echo "-t <string>  -- Test to run. eg: rpc"
+  exit 1
+}
+
+while getopts "h?t" args; do
+case $args in
+    h|\?)
+        usage;
+        exit;;
+    t ) TEST=${OPTARG};;
+  esac
+done
+
+set -euxo pipefail
+
+DATA_DIR=$(mktemp -d -t gossamer-datadir.XXXXX)
+
+if [[ ! "$DATA_DIR" ]]; then
+  echo "Could not create $DATA_DIR"
+  exit 1
+fi
+
+
+# Compile gossamer
+echo "compiling gossamer"
+make build
+
+
+# Run node with static blockchain database
+echo "starting gossamer node in background with http listener on $HOST_RPC"
+
+$PWD/bin/gossamer --port=$PORT --key=$KEY --datadir=$DATA_DIR \
```

Comment:
can you suppress the gossamer logs that get printed to stdout? would be nice to pipe them into another file, it's a bit confusing to run the tests and also see info from the gossamer node

Reply:
hmmm, see the logs is what we want to have when tests break. Pipe it to another file would just make it quite complex to debug results on travis buids as well...